messybrainz/data.py

```diff
@@ -144,11 +144,11 @@ def submit_recording(connection, data):
     Returns:
         the Recording MessyBrainz ID of the data
     """
-    data_json = json.dumps(data, sort_keys=True, separators=(',', ':'))
+    data_json = _convert_to_messybrainz_json(data)
```

Comment:
I wonder if we should only convert the data to lower-case while computing the sha.

In this example, `data_json` is now lower-case and we insert it into the database like that. This means that
1. it's different from what the user submitted
2. if we want to use it to show metadata on the website it's now all lower-case and ugly

as we get the artist name and release data from `data`, the artist and release data will be original case, but the data in the recording table will be lower-cased